frog/imports/ui/LandingPage/index.js

```diff
@@ -20,25 +20,30 @@ const useStyle = makeStyles(theme => ({
     width: '100%',
     maxWidth: '100%',
     padding: theme.spacing(10, 6),
-    color: blueGrey[800],
+    color: blueGrey[800]
+  },
+  introSection: {
+    padding: theme.spacing(12, 0),
```

Comment:
This spacing is quite large, we should make sure it still works on smaller form factors like mobile

Reply:
Changed it to 10, but works well on mobile.